opencog/moses/scoring/bscores.cc

```diff
@@ -225,6 +225,11 @@ void contin_bscore::set_complexity_coef(unsigned alphabet_size, float stdev)
 	                << " complexity ratio = " << 1.0 / _complexity_coef;
 }
 
+Handle contin_bscore::convert_to_atomese(opencog::combo::ComboToAtomeseConverter &converter, const combo_tree &tr) const
```

Comment:
Hmm, I've just realized that `ComboToAtomeseConverter::operator()` is not const, thus prevents you from passing `converter` as const ref. Could you try to make `ComboToAtomeseConverter::operator()` const and then pass `converter` as const ref? (Note: it could be better suited for another PR, on the other hand it is a rather small change that I think it would be acceptable to be including in that PR).

Reply:
Also, line should fit within 80 col.